docs/src/jsx/allLiveEditors.js

```diff
@@ -35,6 +35,36 @@ export const Arrows = makeCodeComponent(require("!!raw-loader!./Arrows"), "Arrow
 
 export const BasicExample = makeCodeComponent(require("!!raw-loader!./BasicExample"), "BasicExample");
 
+export const C11HelloWorld = makeCodeComponent(require("!!raw-loader!./C11HelloWorld"), "RenderingObjects");
```

Comment:
what does C stand for? Maybe it's better to organize these by section, such as put them all in a Tutorial folder

Reply:
C stands for chapter. Will move them all to a tutorial folder. Thought it's still nice to keep the `C11`, `C21` prefix so they are all in order.

Reply:
the number suffix will be a little bit unwieldy if a chapter/subchapter is added before previous ones. Is it necessary to give them an order?

Reply:
I don't have strong opinion about it. Since there are't so many of them right now, I'll just put all under `/tutorials` and remove the prefixes.

Reply:
Thinking about writing more tutorials in the future, so the structure would be something like `/tutorials/duck`, `/tutorials/ballGame` ...